tests/queries/0_stateless/02415_all_new_functions_must_be_documented.reference

```diff
@@ -192,6 +192,7 @@ blockSerializedSize
 blockSize
 buildId
 byteSize
+byteSwap
```

Comment:
All new functions must be documented inline in the code, see `asin.cpp`.

The markdown docs are secondary to the documentation in code, they can contain a copy-paste.

Reply:
Done.